src/Azure/Orleans.Transactions.AzureStorage/Storage/AzureTransactionLogStorage.cs

```diff
@@ -388,13 +398,15 @@ public ArchivalRow(ClusterOptions clusterOptions, byte[] transactions, long firs
             public byte[] Transactions { get; set; }
         }
 
-        private class CommitRow : TableEntity
+        public class CommitRow : TableEntity
         {
-            public CommitRow(long firstLSN, string serviceId)
+            public const string MaxRowKey = "crrk_~";
```

Comment:
crrk = commit row row key?

Reply:
but both partitionkey and rowkey are prefixed. if row key took cr_, what does partitionKey uses? 
come on it is not that ugly.

Reply:
It's ok, just curious. RowKey and PartitionKey can have the same prefix (or no prefix)